.github/workflows/nodejsci.yml

```
@@ -17,13 +17,13 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [ 20.x, 22.x, 24.x ]
+        node-version: [ 22.x, 24.x ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)